bazel/envoy_build_system.bzl

```diff
@@ -139,7 +136,7 @@ def envoy_cc_binary(name,
         visibility = visibility,
         malloc = tcmalloc_external_dep(repository),
         # See above comment on MD5 hash.
-        stamp = 0,
+        stamp = 1,
```

Comment:
What does this change do? How is it different from linkstamp above? Maybe small comment?